flink-runtime-web/web-dashboard/bower.json

```diff
@@ -23,7 +23,8 @@
     "dagre-d3": "~0.4.10",
     "font-awesome": "~4.3.0",
     "moment-duration-format": "~1.3.0",
-    "qtip2": "~2.2.1"
+    "qtip2": "~2.2.1",
+    "highcharts-release": "~4.1.8"
```

Comment:
This library's license is incompatible with the Apache License. Unfortunately, nobody noticed this. We will have to revert the merged changes...